Adding state noteListBar (#15)
inkdropapp/api-docs: src/app/states/note-list-bar/page.mdx

export const metadata = {
  title: 'noteListBar',
  description: 'An app state of the noteListBar. This section explains how to fetch the `noteListBar` state.',
}

# noteListBar

An [app state](/guides/flux-architecture) of the editor.
You can obtain the `noteListBar` state and integrate it with React components.{{ className: 'lead' }}

---

## Data Structure

<Row>
  <Col>
    <Properties>

>       <Property name="selectedNoteIds" type="string[]">

Reviewer: ```suggestion
      <Property name="selectedNoteIds" type="Array<string>">
```

        An array of note ID of selected notes on the note list bar.
      </Property>

>       <Property name="actionTargetNoteIds" type="string[]">

Reviewer: ```suggestion
      <Property name="actionTargetNoteIds" type="Array<string>">
```

>         An array of note ID of selected notes on the note list bar.

Reviewer: ```suggestion
        An array of note IDs of target notes for context menu on the note list bar.
```
